internal/app/networkd/pkg/networkd/networkd.go

```diff
@@ -410,3 +348,52 @@ func (n *Networkd) SetReady() {
 
 	n.ready = true
 }
+
+func (n *Networkd) configureLinks(bonded bool) error {
+	var (
+		err      error
+		errCh    = make(chan error, len(n.Interfaces))
+		multiErr *multierror.Error
+		wg       sync.WaitGroup
+	)
+
+	for _, iface := range n.Interfaces {
+		if iface.Bonded != bonded {
+			continue
+		}
+
+		wg.Add(1)
+
+		go func(netif *nic.NetworkInterface, wg *sync.WaitGroup) {
```

Comment:
usually `wg` is not passed to the closure, as it's not a for loop variable, implicit variable access from outer scope is just fine